src/libraries/System.Text.Json/src/System/Text/Json/JsonHelpers.cs

```diff
@@ -135,5 +136,13 @@ public static bool IsFinite(float value)
             return !(float.IsNaN(value) || float.IsInfinity(value));
 #endif
         }
+
+#if NETSTANDARD2_0 || NETFRAMEWORK
```

Comment:
@layomia What should be here to fix the Windows build?

Reply:
I think we should just revert to just the helper you had earlier. I don't think using this simple built-in public helper is worth the multi-targeting complexity.